docs/source/class_basics.rst

```diff
@@ -232,10 +231,10 @@ effect at runtime:
 Abstract base classes and multiple inheritance
 **********************************************
 
-Mypy supports Python abstract base classes (ABCs). Abstract classes
+Mypy supports Python :doc:`abstract base classes <library/abc>` (ABCs). Abstract classes
```

Comment:
This is a new reference that was added by me, IMO it would be nice to guide the reader to the module document.

Reply:
+1